src/main/java/frc/robot/commands/DefaultDriveCommand.java

```diff
@@ -28,7 +30,7 @@ public void execute() {
     double ySpeed = m_driver.getSideTranslation();
     double rot = m_driver.getRotation();
 
-    m_drive.drive(xSpeed, ySpeed, rot, true);
+    m_drive.drive(DriverStation.getAlliance()==Alliance.Blue?xSpeed:-xSpeed, DriverStation.getAlliance()==Alliance.Blue?ySpeed:-ySpeed, rot, true);
```

Comment:
This wouldn't be necessary because the pigeon would be reset to be facing the other direction when you are on the other alliance. It's already kind of done by the constant kStartingHeading, or something like that.
But beyond that, I would like to note two things:
1) I would prefer this be done somehow in the init()
2) This code has nothing to do with vision, so it shouldn't be on this branch/PR

Reply:
I added it because the controls were inverted for testing, but I'll change it back.